flutter/test/load_android_image_list_test.dart

```diff
@@ -125,3 +132,21 @@ SentryEvent getEvent({bool symbolicated = false}) {
   );
   return SentryEvent(exception: ex);
 }
+
+class Fixture {
+  Fixture() {
+    channel = MethodChannel('sentry_flutter');
+    options = SentryFlutterOptions()..dsn = fakeDsn;
+    hub = Hub(options);
```

Comment:
could `Hub` be a mock instead or the real impl?

Reply:
Nope, i tried that and the tests failed. Some tests need the event processor to get executed which the mock does not have.